ons-spark/raw-notebooks/checkpoint-staging/r_input.R

```diff
@@ -0,0 +1,155 @@
+options(warn = -1) 
+library(sparklyr)
+library(dplyr)
+library(DBI)
+
+sc <- sparklyr::spark_connect(
+    master = "local[2]",
+    app_name = "checkpoint",
+    config = sparklyr::spark_config())
+
+
+set.seed(42)
+new_cols <- 12
+num_rows <- 10^3
+
+start_time <- Sys.time()
+
+df = sparklyr::sdf_seq(sc, 1, num_rows) %>%
+    sparklyr::mutate(col_0 = ceiling(rand()*new_cols))
+
+for (i in 1: new_cols)
+{
+  column_name = paste0('col_', i)
+  prev_column = paste0('col_', i-1)
+  df <- df %>%
+    sparklyr::mutate(
+      !!column_name := case_when(
+        !!as.symbol(prev_column) > i ~ !!as.symbol(prev_column)))
+  
+}
+
+df %>%
+    head(10)%>%
```

Comment:
Another minor stylistic point - should be a space before %>% (also occurs on line beginning 78) - again apologies if too pedantic!